docs/source/pages/DM9098.rst

```diff
@@ -0,0 +1,126 @@
+.. _dm9098:
+
+OAK-D-Pro
+=========
+
+(`shop <https://shop.luxonis.com/collections/beta-store/products/oak-d-pro>`__)
+
+.. image:: /_static/images/DM9098/oak-d-pro-front.png
+
+Overview
+********
+
+The OAK-D-Pro is an upgraded version of :ref:`OAK-D`, featuring **IR laser dot projector** (active stereo) and **IR illumination LED** (for
+"night vision"). It's also smaller and lighter than the OAK-D.
+It's currenty in **beta stage** so you have to use a custom version of the depthai library to communicate with this device,
+more information below at *Getting started*.
+
+How it works
+************
+
+OAK-D-Pro has **notch IR filters** (940nm) on mono cameras, which allows only visible light and IR light from illumination LED/laser dot
```

Comment:
Notch is not implemented with first samples and first production build, CCMs are only IR capable or w/out IR filter. 
We plan to use CCMs with notch filter in future builds though.